vignettes/Rcpp-modules.Rmd

```diff
@@ -1018,6 +1085,14 @@ Rcpp::NumericVector vec_asR( vec* obj ) {
 void vec_set(vec* obj, int i, double value) { 
    obj->at( i ) = value; 
 }
+// CHECK-PR: workaround complilation error with vecL:resize (C++11)
```

Comment:
I'll try to investigate this a bit later.

Reply:
I actually just realized this fix was part is part of one of the skeleton files since a long time https://github.com/RcppCore/Rcpp/blob/4ec861775601ab9c6fcad3fd03d4b84ba88814b5/inst/skeleton/stdVector.cpp#L45-L46